oneplus/com_views.py

```diff
@@ -273,6 +273,7 @@ def post():
 
                 _message.save()
             request.session["state"]["chat_page_max"] += 1
+            return redirect("com.chat", chatid=chatid)
```

Comment:
This was necessary, yes. It fixes an issue on the same page, where the user would sometimes see a message twice.